content/en/logs/processing/parsing.md

```diff
@@ -247,27 +248,46 @@ Other examples:
 **Multiple QuotingString example**: When multiple quotingstring are defined, the default behavior is replaced with a defined quoting character.
 The key-value always matches inputs without any quoting characters, regardless of what is specified in `quotingStr`. When quoting characters are used, the `characterWhiteList` is ignored as everything between the quoting characters is extracted.
 
-Log:
+**Log:**
 
   ```text
   key1:=valueStr key2:=</valueStr2> key3:="valueStr3"
   ```
 
-Rule:
+**Rule:**
 
   ```text
   rule %{data::keyvalue(":=","","<>")}
   ```
 
-Result:
+**Result:**
 
   ```json
-  {
-    "key1": "valueStr",
-    "key2": "/valueStr2"
-  }
+  {"key1": "valueStr", "key2": "/valueStr2"}
   ```
-  
+**Custom delimiter example**: Define a custom delimiter if all the key values pairs are concatenated with no standard delimiter to split them. Note that defining `""` as `quotingStr` keeps the default configuration for quoting.
```

Comment:
Why do we mention quotingStr here?

Reply:
As the user need to define the third parameter (quoting string) to be able to define the delimiter even if he doesn't want to change anything the the quoting string, I found it less miss-leading to explicitly explain  how deal with it as it should be a very common case.
Can remove if you think it's unnecessary, but I would prefer to keep it.

Reply:
removed